index.html

```diff
@@ -2,12 +2,314 @@
 <html lang="en">
   <head>
     <meta charset="utf-8" />
+    <meta http-equiv="X-UA-Compatible" content="IE=edge" />
+    <meta name="viewport" content="width=device-width, initial-scale=1" />
+    <title>My Portfolio</title>
+    <link rel="shortcut icon" href="favicon.ico" type="image/x-icon" />
+    <link rel="icon" href="favicon.ico" type="image/x-icon" />
+    <link
+      href="https://fonts.googleapis.com/css?family=Lato:300,400,700,900"
+      rel="stylesheet"
+    />
 
-    <link href="styles.css" rel="stylesheet" />
-
-    <title>Rocket Academy</title>
+    <link href="./styles.css" rel="stylesheet" />
   </head>
+
   <body>
-    <h1>Hello, world!</h1>
+    <header>
+      <div class="nav-bar">
+        <ul class="nav-button-wrapper">
+          <li>
+            <a href="#about">About</a>
+          </li>
+          <li>
+            <a href="#experience">Experience</a>
+          </li>
+          <li>
+            <a href="#education">Education</a>
+          </li>
+          <li>
+            <a href="#projects">Projects</a>
+          </li>
+          <li>
+            <a href="#skills">Skills</a>
+          </li>
+          <li>
+            <a href="#contact">Contact</a>
+          </li>
+        </ul>
+      </div>
+    </header>
+    <div class="section-wrapper">
+      <div id="lead" class="section">
+        <div id="lead-content">
+          <h1>Eugene Ng</h1>
+          <h2>Software Engineer</h2>
+          <a href="#" class="btn-rounded-white">Download Resume</a>
+        </div>
+        <!-- End #lead-content -->
+
+        <div id="lead-down">
+          <span>
+            <i class="fa fa-chevron-down" aria-hidden="true"></i>
+          </span>
+        </div>
+        <!-- End #lead-down -->
+      </div>
+      <!-- End #lead -->
+
+      <div id="about" class="section background">
+        <div class="col-md-4">
+          <h2 class="heading">About Me</h2>
+        </div>
+        <div class="col-md-8">
+          <p>
+            Lorem ipsum dolor sit amet, consectetur adipiscing elit. Curabitur
+            in iaculis ex. Etiam volutpat laoreet urna. Morbi ut tortor nec
+            nulla commodo malesuada sit amet vel lacus. Fusce eget efficitur
+            libero. Morbi dapibus porta quam laoreet placerat. Donec eu vehicula
+            neque. Donec viverra lorem nunc, tempus euismod eros sollicitudin
+            ut. Quisque et tincidunt libero. Donec id pharetra justo. Proin
+            euismod lacinia dolor, eu scelerisque justo tempus pharetra. Vivamus
+            nunc justo, finibus ut nisl sed, euismod rhoncus nulla. Proin sed
+            magna egestas, egestas ante et, congue eros. In consequat, mauris
+            dapibus tincidunt suscipit, ex libero aliquet diam, at maximus risus
+            enim non leo.
+          </p>
+        </div>
+      </div>
+      <!-- End #about -->
+
+      <div id="experience" class="section background-alt">
```

Comment:
good that you're mixing up ids and multiple classes. this is a great way to create small styles and not load everything into a single class/id